arm/servicebus/models.go

```diff
@@ -453,29 +375,24 @@ func (client TopicListResult) TopicListResultPreparer() (*http.Request, error) {
 		autorest.WithBaseURL(to.String(client.NextLink)))
 }
 
-// TopicProperties is
+// TopicProperties is the Tpoic Properties definition.
 type TopicProperties struct {
-	AccessedAt                          *date.Time               `json:"accessedAt,omitempty"`
-	AutoDeleteOnIdle                    *string                  `json:"autoDeleteOnIdle,omitempty"`
-	EntityAvailabilityStatus            EntityAvailabilityStatus `json:"entityAvailabilityStatus ,omitempty"`
-	CreatedAt                           *date.Time               `json:"createdAt,omitempty"`
-	CountDetails                        *MessageCountDetails     `json:"countDetails,omitempty"`
-	DefaultMessageTimeToLive            *string                  `json:"defaultMessageTimeToLive,omitempty"`
-	DuplicateDetectionHistoryTimeWindow *string                  `json:"duplicateDetectionHistoryTimeWindow ,omitempty"`
-	EnableBatchedOperations             *bool                    `json:"enableBatchedOperations,omitempty"`
-	EnableExpress                       *bool                    `json:"enableExpress,omitempty"`
-	EnablePartitioning                  *bool                    `json:"enablePartitioning,omitempty"`
-	EnableSubscriptionPartitioning      *bool                    `json:"enableSubscriptionPartitioning,omitempty"`
-	FilteringMessagesBeforePublishing   *bool                    `json:"filteringMessagesBeforePublishing,omitempty"`
```

Comment:
We're currently exposing this field, but were unaware it was Deprecated / scheduled to be removed.

I can see [it's been removed from the Swagger in this commit](https://github.com/v-Ajnava/azure-rest-api-specs/commit/5d6a17103c57e57b2b5e49e77ec118b684f8366e#diff-654d402701463525866911e60a6308a1L2443) - however presumably it's still being returned in the API for the moment, is that likely to change?